README.md

```diff
@@ -8,13 +8,15 @@
 [![Link: NuGet](https://img.shields.io/badge/NuGet-633b9b?style=flat-square&logo=nuget&labelColor=gray)](https://www.nuget.org/profiles/Esri_Inc)
 [![Link: Esri Community](https://img.shields.io/badge/🙋-Get%20help%20in%20Esri%20Community-633b9b?style=flat-square)](https://community.esri.com/t5/arcgis-runtime-sdk-for-net/bd-p/arcgis-runtime-sdk-dotnet-questions)
 
-# ArcGIS Runtime SDK for .NET - Samples
+# ArcGIS Runtime SDK for .NET - Samples (200.0 Beta)
 
 <a href="//www.microsoft.com/store/apps/9mtp5013343h?cid=storebadge&ocid=badge"><img src="./samples_screenshot.png" title="Get the viewer from Microsoft" alt="Screenshot of the sample viewer for WPF" width="500px" /></a>
 <hr />
 
 **Interactive samples demonstrate the ArcGIS Runtime API**
 
+> **WELCOME TO THE 200.0 BETA**: ArcGIS Runtime 200.0 beta samples are available for WPF, WinUI, and UWP. We have some MAUI samples ready for the beta, and many more will be available with the full launch of 200.0. Xamarin content has been removed. Xamarin is only supported in ArcGISRuntime 100.15 versions. These samples are still availble in the [`main branch`](https://github.com/Esri/arcgis-runtime-samples-dotnet) of the repository and will continue to be available in a 100.15 branch with the release of 200.0.
```

Comment:
```suggestion
> **Note**:  This branch is under ArcGIS Runtime 200.0.0-beta program and showcases changes we will be releasing in our upcoming 200.0.0 release. Sample are available for WPF, WinUI, and UWP. We have some MAUI samples ready for the beta, and many more will be available with the final release of 200.0. Please join the [Esri Early Adopter community](https://esriurl.com/agsrt200beta) to provide feedback and report issues.
Also note, Xamarin content has been removed since Xamarin is only supported until ArcGISRuntime 100.15 versions. These samples are still available in the [`main branch`](https://github.com/Esri/arcgis-runtime-samples-dotnet) of the repository and will continue to be available in a 100.15 branch with the release of 200.0.
```